sdk/typescript/src/types/events.ts

```diff
@@ -129,9 +129,15 @@ export type SuiEventFilter =
   | { And: [SuiEventFilter, SuiEventFilter] }
   | { Or: [SuiEventFilter, SuiEventFilter] };
 
+export type SuiEventId = {
+  tx_seq_num: number;
```

Comment:
We should do one of the following:
1) standardize the "number" suffix - either `num` or `number` for both
2) eliminate the suffix

Reply:
Ah sure.  Does it matter that the naming isn't consistent with the Rust?   Also seems like it should be camelCase, I'll fix that